### PR TITLE
Astech Divide By Zero Errors

### PR DESCRIPTION
**Pull request by Windchild292**
This prevents a few divide by zero issues raised on the Discord while improving the code for Campaign::getTargetFor and Campaign::getAvailableAstechs and adding a constant for Astech team size.

### CODE REVIEW

File: MekHQ/src/mekhq/campaign/Campaign.java [hunk 1]
```diff
@@ -4847,37 +4842,38 @@ public TargetRoll getTargetFor(IPartWork partWork, Person tech) {
             target.addModifier(getFaction().getEraMod(getGameYear()), "era");
         }
 
-        boolean isOvertime = false;
+        final boolean isOvertime;
```
Comment (NickAragua) [inline MekHQ/src/mekhq/campaign/Campaign.java:4845]: What was wrong with the way it was before? Initializing a variable to its default state is a perfectly valid coding pattern.

File: MekHQ/src/mekhq/campaign/Campaign.java [hunk 2]
```diff
@@ -4847,37 +4842,38 @@ public TargetRoll getTargetFor(IPartWork partWork, Person tech) {
...
-        if (null != partWork.getUnit() && partWork.getUnit().isSelfCrewed()) {
-            int hits;
-            if (null != partWork.getUnit().getEntity().getCrew()) {
-                hits = partWork.getUnit().getEntity().getCrew().getHits();
-            } else {
-                hits = 6;
-            }
-            helpMod = getShorthandedModForCrews(hits);
+        if ((partWork.getUnit() != null) && partWork.getUnit().isSelfCrewed()) {
+            helpMod = getShorthandedModForCrews((partWork.getUnit().getEntity().getCrew() == null)
```
Comment (NickAragua) [inline MekHQ/src/mekhq/campaign/Campaign.java:4862]: This looks familiar, didn't I see the same logic in the other pull request I just reviewed?

Maybe a better solution would be to unify that logic somewhere.

**Review by NickAragua — CHANGES_REQUESTED** [review]

File: MekHQ/src/mekhq/campaign/Campaign.java (hunk 2 above)
Comment (Windchild292) [reply MekHQ/src/mekhq/campaign/Campaign.java:4862]: You did, and I've got a change that standardizes the logic in that PR.

File: MekHQ/src/mekhq/campaign/Campaign.java (hunk 1 above)
Comment (Windchild292) [reply MekHQ/src/mekhq/campaign/Campaign.java:4845]: This is a way to handle it that means you know the value is set here and never changes. I find it easier when debugging, as I don't have to trace down any other uses.

**Review by NickAragua — APPROVED** [review]

**Pull request merged**